connectors/utils.py

```diff
@@ -760,29 +769,29 @@ async def extract_with_file_send(self, filepath, filename):
 
         Returns a parsed response
         """
-        with open(filepath, "rb") as file:
-            file_data = aiohttp.FormData()
-            file_data.add_field("file", BytesIO(file.read()), filename=filename)
-
-            async with self._begin_session().post(
-                f"{self.host}/extract_text/", data=file_data
-            ) as response:
-                return await self.parse_extraction_resp(filename, response)
+        async with self._begin_session().put(
+            f"{self.host}/extract_text/", data=self.file_sender(filepath)
+        ) as response:
+            return await self.parse_extraction_resp(filename, response)
+
+    async def file_sender(self, filepath):
+        async with aiofiles.open(filepath, "rb") as f:
+            chunk = await f.read(64 * 1024)
```

Comment:
Is it 64 kilobytes?

Reply:
I think it's nice to make this setting configurable via the initializer or configuration

Reply:
> Is it 64 kilobytes?

It is!

> I think it's nice to make this setting configurable via the initializer or configuration

Good idea, I'll add that

Reply:
@artem-shelkovnikov added!

Reply:
It would be under `extraction_service.stream_chunk_size` in the config now.
(config is optional for this so I haven't added it to the yml file).